ext/web/lib.rs

```diff
@@ -314,46 +316,29 @@ impl Resource for TextDecoderResource {
   }
 }
 
-#[derive(Serialize)]
-#[serde(rename_all = "camelCase")]
-struct EncodeIntoResult {
-  read: usize,
-  written: usize,
-}
```

Comment:
The original code used JS floats, the code you're updating from used `usize`, the WebIDL uses `u64` (<https://encoding.spec.whatwg.org/#dictdef-textencoderencodeintoresult>), but the new implementation uses `u32`?
This may truncate results from (absurdly) long strings, if that's even a possible case.

Reply:
> if that's even a possible case.

This is worth revisiting when V8 string limit is bumped > 4GB. Right now, Node.js doesn't care either https://github.com/nodejs/node/blob/5fad0b93667ffc6e4def52996b9529ac99b26319/lib/internal/encoding.js#L346

Reply:
Ah, great to see that it's not an issue then!